.obs/dockerfile/slem-base-os/Dockerfile

```diff
@@ -1,8 +1,9 @@
 # SPDX-License-Identifier: Apache-2.0
 # Define the names/tags of the container
-#!BuildTag: suse/sle-micro/base-%%SLEMICRO_VERSION%%:latest
-#!BuildTag: suse/sle-micro/base-%%SLEMICRO_VERSION%%:%VERSION%
-#!BuildTag: suse/sle-micro/base-%%SLEMICRO_VERSION%%:%VERSION%-%RELEASE%
+#!BuildName: SLE-Micro-base-container
+#!BuildTag: suse/sle-micro/sle-micro-base-container-%%SLEMICRO_VERSION%%:latest
+#!BuildTag: suse/sle-micro/sle-micro-base-container-%%SLEMICRO_VERSION%%:%VERSION%
+#!BuildTag: suse/sle-micro/sle-micro-base-container-%%SLEMICRO_VERSION%%:%VERSION%-%RELEASE%
```

Comment:
Is there any specific reason to drop the `/` from the tag. This is likely to cause issues in the registry as these paths are going through some path filters to determine its accessibility.

Reply:
Not sure what you're refering to _exactly_ 🤔